packages/manager/apps/dedicated/client/app/dedicatedCloud/license/enable/dedicatedCloud-license-enable.html

```diff
@@ -43,6 +43,7 @@
                 </oui-datagrid>
 
                 <div
+                    data-ng-if="$ctrl.bindings.contracts.value.length > 0"
```

Comment:
```suggestion
                    data-ng-if="!$ctrl.allContractsSigned"
```
and define and initialize allContractsSigned in the controller